Jenkinsfile_CNP

```diff
@@ -18,6 +18,15 @@ def nonPrSecrets = [
   'bulk-scan-${env}': [
     secret('notifications-queue-listen-connection-string', 'NOTIFICATION_QUEUE_CONN_STRING_READ'),
     secret('notifications-queue-send-connection-string', 'NOTIFICATION_QUEUE_CONN_STRING_WRITE')
+  ],
+  's2s-${env}': [
+    secret('microservicekey-bulk-scan-processor-tests', 'TEST_S2S_SECRET')
```

Comment:
chose this as this service mainly will be pushing notifications and notification service itself should not need any s2s token at all - it is only validating/extracting the requester service name from auth token header.

plus it is not rely as such - just relevant to us s2s token ;). real service will be using real keys which we all know what they are / will be

bottom line - don't want to create extra secrets for functional tests for notification service which it doesn't need. use existing ones

Reply:
blob router orchestrator and potentially crime would publish to notification service. Also if the secret for processor is changed then this service would also needs deployment flux changes.
So it would be better to use it's own secrets

Reply:
services can change secrets as many times as they want. note: `bulk-scan-processor-tests`. **tests**. not use in real life. as far as i remember platform was mocking about having all those tests and what not individually but i can create if it sets out mind at peace. just saying that for tests if we wish we could just use one. e.g. like sample app making all those requests everywhere. as it is dummy app anyway could just use for functional testing as well

Reply:
ok not a major issue if we can please add a comment that processor is consumer of notification that would be helpful for anyone looking at code